etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/etor/ruleengine/transformation/custom/MapLocalObservationCodesTest.groovy

```diff
@@ -0,0 +1,322 @@
+package gov.hhs.cdc.trustedintermediary.etor.ruleengine.transformation.custom
+
+import gov.hhs.cdc.trustedintermediary.ExamplesHelper
+import gov.hhs.cdc.trustedintermediary.context.TestApplicationContext
+import gov.hhs.cdc.trustedintermediary.external.hapi.HapiFhirHelper
+import gov.hhs.cdc.trustedintermediary.external.hapi.HapiFhirResource
+import gov.hhs.cdc.trustedintermediary.external.hapi.HapiHelper
+import gov.hhs.cdc.trustedintermediary.wrappers.Logger
+import org.hl7.fhir.r4.model.Bundle
+import org.hl7.fhir.r4.model.Coding
+import org.hl7.fhir.r4.model.Observation
+import org.hl7.fhir.r4.model.StringType
+import spock.lang.Specification
+
+class MapLocalObservationCodesTest extends Specification {
+    def transformClass
+    def mockLogger = Mock(Logger)
+
+    def setup() {
+        TestApplicationContext.reset()
+        TestApplicationContext.init()
+        TestApplicationContext.injectRegisteredImplementations()
+        TestApplicationContext.register(Logger, mockLogger)
```

Comment:
You should register first, then inject.  The current order doesn't inject the logger

Reply:
Done